File: uri/1 - Basics/1009/Main.c
#include <stdio.h>

int main(){
    double f1, f2, f3;
    char name[256];

    scanf(" %[^\n]",name);
    scanf("%lf", &f1);
    scanf("%lf", &f2);
    f3 = f1 + (15.0*f2/100.0);
    printf("TOTAL = R$ %.2f\n",f3 );

    return 0;
}

File: uri/1 - Basics/1009/Main.go
package main

import "fmt"

func main() {
	var f1, f2, f3 float64
	var name string
	fmt.Scanf("%s", &name)
	fmt.Scanf("%f", &f1)
	fmt.Scanf("%f", &f2)
	f3 = f1 + (15.0 * f2 / 100.0)
	fmt.Printf("TOTAL = R$ %.2f\n", f3)
}
